src/lib.rs

```diff
@@ -146,7 +146,7 @@ impl<'a> Image<'a> {
 						});
 					}
 					ColorMode::Rgb => {
-						if false {
+						if y & 1 == 0 {
```

Comment:
I think `y % 2` is a bit better. Also, a little concerned that this probably breaks Luma ColorMode?

Reply:
I'm happy with `y & 1`. @dmadisetti what's the concern with Luma ColorModel here?